src/ros_vrpn_client.cpp

```diff
@@ -234,25 +294,18 @@ int main(int argc, char* argv[])
   private_nh.param<std::string>("vrpn_server_ip", vrpn_server_ip, std::string());
   private_nh.param<int>("vrpn_port", vrpn_port, 3883);
   private_nh.param<std::string>("vrpn_coordinate_system", coordinate_system_string, "vicon");
-  private_nh.param<std::string>("object_name", object_name, "bluebird");
+  private_nh.param<std::string>("object_name", object_name, "auk");
```

Comment:
I just noticed the default for this param in the launch file was auk and thought I'd make the default here consistent with that. But yeah doesn't matter.